lib/Client.php

```diff
@@ -336,30 +338,32 @@ protected function doRequest(RequestInterface $request): ResponseInterface
      * By keeping this resource around for the lifetime of this object, things
      * like persistent connections are possible.
      *
-     * @var resource
+     * @var resource|null
      */
-    private $curlHandle;
+    private $curlHandle = null;
 
     /**
      * Handler for curl_multi requests.
      *
      * The first time sendAsync is used, this will be created.
      *
-     * @var resource
+     * @var resource|null
      */
-    private $curlMultiHandle;
+    private $curlMultiHandle = null;
```

Comment:
This helps phpstan to process potential access to $curlMultiHandle before it is written (checks for `=== null` are seen as valid)